src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "***John***";
+        Task[] tasks = new Task[100];
+        String userInput = "";
+        int count = 0;
+
+        displayWelcome(logo);
+
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
+            userInput = in.nextLine();
+            if (userInput.equals("bye")){
+                break;
+            }
+            else if (userInput.equals("list")){
+                listTasks(tasks, count);
+            }
+            else if (userInput.startsWith("done")){
+                markTaskAsDone(tasks, userInput);
+            }
+            else if (userInput.startsWith("todo")){
+                count = addTodoTask(userInput, tasks, count);
+            }
+            else if (userInput.startsWith("deadline")){
+                count = addDeadlineTask(userInput, tasks, count);
+            }
+            else if (userInput.startsWith("event")){
+                count = addEventTask(userInput, tasks, count);
+            }
+            else {
+                count = addGeneralTask(userInput, tasks, count);
+            }
+        }
+        System.out.println("Bye. Hope to see you soon!");
+    }
+
```

Comment:
Maybe you can summarise this in a method in order to add SLAP to your main function.

Reply:
There is a glaring coding standard violation in this code :-p

Reply:
Yes, I've noticed that I have made a mistake in the coding standard. Thanks for your input!